### ISSUE TITLE
Trainloop

### ISSUE THREAD

**Issue by tonislees** [kevinkorrol/DeepFork#22]
Implement a class for PyTorch's DataLoader and Dataset.

Implement a class for a loss function for neural net where it compares neural net output and expected outputs from the sample. The logic for comparing the move distribution and move that was made needs to be thought through.

Implement a function, that uses the data loading class, for training the model. It should take input data from data/processed and run for n epochs.
